client/js/components/faq/DpFaqSupport.vue

```diff
@@ -0,0 +1,121 @@
+<license>
+(c) 2010-present DEMOS plan GmbH.
+
+This file is part of the package demosplan,
+for more information see the license file.
+
+All rights reserved
+</license>
+
+<template>
+  <div
+    id="scFaq"
+    class="c-support-wrapper space-inset-m">
+    <h2 class="font-normal">
+      {{ Translator.trans('support.heading') }}
+    </h2>
+    <section class="mb-5">
+      <p class="mb-5">
+        {{ Translator.trans('support.introduction') }}
+      </p>
+      <p>
+        Sie haben eine Frage zur Technik? Dann hilft Ihnen unser technischer Support gerne weiter.
+      </p>
+        <h3 class="font-size-h4 mt-5">
+          {{ Translator.trans('support.content') }}
+        </h3>
+      <ul class="grid lg:grid-cols-3 gap-3">
+        <li
+          v-for="contact in contacts"
+          class="space-inset-m h-48 c-support-singlecard">
+          <dp-faq-support-card>
+            <template v-slot:title>
+              <h2 class="font-semibold">
```

Comment:
it is not a good practice to use an `h2` heading immediately after an `h3` heading in the hierarchical order. It make harder for assistive technologies to understand the document's structure and hierarchy.

I think H4 is a good choice for this reason because it has a font size of 18px, so you won't need to change it later in the `.c-support__groupcard & > div > h2`

Reply:
done https://github.com/demos-europe/demosplan-core/pull/2139/commits/98c4a599f6ab47069611f3c0b89242c2728839e7